src/Controller/AddOnOptionsController.py

```diff
@@ -147,7 +147,7 @@ def open_link(self, item):
     ######################################################################################################
     # windowing
     @Slot(QtCore.QPoint)
-    def on_customContextMenuRequested_Window(self, pos):
+    def on_customContextMenuRequestedWindow(self, pos):
```

Comment:
But I don't see the change in signature other than the renaming, nor a change in annotation

Reply:
By renaming the slots (removing the last underscore) it solves the issue.
The next question would be is this appropriate? 
For example would onCustomContextMenuRequestedWindow be more appropriate than  on_customContextMenuRequestedWindow?

Reply:
I'm still learning this - Any input you have to further my understanding is more than welcome!

Reply:
The key thing is that the renaming is the fix.
I don't know anything about the signal and slot naming conventions.   I think Matthew is our local expert.

Reply:
> I'm still learning this - Any input you have to further my understanding is more than welcome! 

That's what code review is good for.
We know that the teams are going through a learning curve and the biggest contribution I can make is helping individuals through the learning curve.

Reply:
Thank you Stuart, it is well appreciated, and will continue to be appreciated throughout the year. Cheers!